apis/ec2/v1beta1/vpc_types.go

```diff
@@ -81,6 +81,20 @@ type VPCParameters struct {
 	// +immutable
 	CIDRBlock string `json:"cidrBlock"`
 
+	// The IPv6 CIDR block from the IPv6 address pool. You must also specify Ipv6Pool
```

Comment:
one last question `You must also specify Ipv6Pool in the request` did we really need Ipv6Pool for this Input ? if yes think it is missing / if not - we need to remove this comment

Reply:
Yes, also added Ipv6Pool.